.github/workflows/security_tests_v2.yml

```diff
@@ -0,0 +1,189 @@
+name: V2 Security Tests
+
+on:
+  push:
+    branches: [ v2 ]
+  workflow_dispatch:
+    inputs:
+      docker-tag:
+        description: 'Docker tag for scan'
+        default: 'latest'
+        required: false
+
+jobs:
+  security-operator:
+    runs-on: ubuntu-latest
+    container: snyk/snyk:golang
+    steps:
+    - uses: actions/checkout@v2
+    - name: security-golang
+      env:
+        SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+      run: |
+        make -C operator fmt
+        snyk test --file=operator/go.mod --fail-on=upgradable --severity-threshold=high
+
+  security-scheduler:
+    runs-on: ubuntu-latest
+    container: snyk/snyk:golang
+    steps:
+    - uses: actions/checkout@v2
+    - name: security-golang
+      env:
+        SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+      run: |
+        make -C scheduler lint
+        snyk test --file=scheduler/go.mod --fail-on=upgradable --severity-threshold=high
+
+  security-image-operator:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set default docker tag for builds from v2 branch
+      id: docker-tag
+      run: |
+        USER_INPUT="${{ github.event.inputs.docker-tag }}"
+        echo ::set-output name=value::${USER_INPUT:-"latest"}
```

Comment:
Oh wow - is this wizardry to default to `latest` if no tag is present?

Reply:
Yeah, that's to use default for input usually coming as parameter to "on_dispatch" in workflows running "on_push". There may be more elegant way (though I doubt, unless some new features were added)  but that is what I found when working on SCv2 release workflow.